exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java

```diff
@@ -211,16 +211,25 @@ public String getExecutionDuration() {
       return NOT_AVAILABLE_LABEL;
     }
 
+    long queryEndTime;
+
+    // Check if State is RUNNING, set end time to current time
+    if (profile.getState() == QueryState.RUNNING) {
```

Comment:
@prasadns14  I think that after rebase you have lost one of the initial changes. Since you did not remove check in previous if statement for `profile.getState() == QueryState.RUNNING`, this if statement will never be executed.

Reply:
@arina-ielchiieva, thank you for catching the mistake.
Fixed it